src/features/filters/CommentsFilter.jsx

```diff
@@ -40,7 +40,9 @@ export const CommentsFilter = () => {
                   name="filter"
                   id="filter"
                   component="textarea"
-                  placeholder=""
+                  placeholder="Comments with the phrase..."
+                  onKeyDown={(e) => e.stopPropagation()}
```

Comment:
This fixes a 'bug' where keyboard navigation would work even when entering text in the text area.  The reproduction steps are
1. Open the comments popover
2. Create a new comment
3. Type in the comment filter text area
4. Check that keyboard navigation is working